src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/AllMetrics.java

```diff
@@ -32,6 +32,7 @@
   // metric name (not complete, only metrics use the json format and contains
   // numeric values. Will add more when needed)
   public enum MetricName {
+    CACHE_CUSTOM,
```

Comment:
Minor: The naming is not very intuitive for me. Should this just be `CACHE METRICS`? Maybe I'm missing the context around how it is intended to be used.

Reply:
Adding the writer side PR for more context : https://github.com/opendistro-for-elasticsearch/performance-analyzer/pull/145

The idea is that cache max size is a node level static setting, unlike cache hit, miss, size and eviction (part of SHARD_STATS) which are per shard and thus, the two category cannot be clubbed together and  `CACHE METRICS` being the generic name cannot be used. 

I agree `CACHE_CUSTOM` isnt very explanatory and I am open to better naming suggestions here.

Reply:
Thanks for the pointers. How about `CACHE_CONFIG`, since this has cache configuration (max_size etc.) ?

Reply:
can we simple call it CACHE ? There is also a THREAD_POOL metric down below. I think we can simply call it CACHE. 

And why can't we add this directly into SHARD_STATS metric ?

Reply:
Since maxSize metric isn't collected at shard level but at node level, and `SHARD_STATS` uses `ShardID, IndexName` as dimensions, it would be confusing for the end users if this metric was added under `SHARD_STATS`.

I will rename `CACHE_CUSTOM` to `CACHE_CONFIG`.